### PR TITLE
Improve support for TRC files

### PR DESCRIPTION
**Pull request by twam**
* Add support for Version 2.0
* Add support for $STARTTIME in Version 1.1 and Version 2.1
* Add support for $COLUMNS in Version 2.1

### CODE REVIEW

**Comment by zariiii9003** [comment]
@pkess could you take a look?

**Comment by pkess** [comment]
Hello @zariiii9003, hello @twam 
Thank you for your request for a review from me. I had a deeper look into this now and the changes are looking great. Thank you for this contribution.

It might be nice to add some more tests and test data in the future but this would improve this package very much just as it is right now

File: can/io/trc.py [hunk 1]
```diff
@@ -122,15 +159,38 @@ def _parse_msg_V1_1(self, cols: List[str]) -> Optional[Message]:
         msg.is_rx = cols[2] == "Rx"
         return msg
 
-    def _parse_msg_V2_1(self, cols: List[str]) -> Optional[Message]:
+    def _parse_msg_V2_x(self, cols):
```
[the remark this answers is not in the document]
Comment (twam) [reply can/io/trc.py]: Added.

**Review by zariiii9003 — APPROVED** [review]

**Comment by zariiii9003** [comment]
Thanks!

**Pull request merged**